### DISCUSSION TITLE
Suitability for CAN-Bridging?

### DISCUSSION THREAD

**Discussion by mxmxchere** [c3re/can2mqtt#76] [Q&A]

**Comment by zakozafer** [comment]
hello
what i want to do in the project
i want to send canbus data to esp32 with the server and mpc2515 connected to the canbus raspberry on the car and convert it to canbus with the mpc2515 connected to esp32 is it possible to do this with your project?
i tried to do what i want to do with raspberry and esp32 mosuitto unfortunately i couldn't do it?
actually i just want to do is connect the canbus on the car to the interface device on the desktop, receive canbus and convert it back to canbus from the internet and provide remote access with a device like elm327

**Comment by mxmxchere** [comment]
Hi @zakozafer thanks for your question. I am not exactly sure if I understood your question correctly. But I will try my best to answer. Generally speaking you can use can2mqtt whenever you are running something with a SocketCAN Interface. This is almost always a Linux system. A raspberry Pi with a MPC2515 works very well. Can2mqtt is supposed to transfer data towards (and from) a MQTT Broker. What happens outside of this scope (Subscription with an ESP32 and conversion back to CAN-Bus) is not part of this Project (do one thing and do it well).
